### PR TITLE
qa/tasks: Fix ambiguous store_thrash, thrash_store

### PR DESCRIPTION
**Pull request by joscollin**
Fixes: http://tracker.ceph.com/issues/39159

### CODE REVIEW

File: qa/tasks/mon_thrash.py [hunk 1]
```diff
@@ -70,11 +70,11 @@ class MonitorThrasher:
     - mon_thrash:
         revive_delay: 20
         thrash_delay: 1
-        thrash_store: true
+        thrash_store: True
```
Comment (batrick) [inline qa/tasks/mon_thrash.py]: YAML is case insensitive for booleans. This change is not necessary.

File: qa/tasks/mon_thrash.py [hunk 2]
```diff
@@ -70,11 +70,11 @@ class MonitorThrasher:
...
         thrash_store_probability: 40
         seed: 31337
-        maintain_quorum: true
-        thrash_many: true
+        maintain_quorum: True
+        thrash_many: True
```
Comment (batrick) [inline qa/tasks/mon_thrash.py]: ditto

**Comment by batrick** [comment]
@neha-ojha please include in next rados run.

**Comment by liewegas** [comment]
http://tracker.ceph.com/issues/39394

thrash_store is both a bool member and a function name!

File: qa/tasks/mon_thrash.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ class MonitorThrasher:
     freeze_mon_duration: how many seconds to freeze the mon (default: 15)
     scrub               Scrub after each iteration (default: True)
 
-    Note: if 'store-thrash' is set to True, then 'maintain-quorum' must also
```
Comment (neha-ojha) [inline qa/tasks/mon_thrash.py]: wouldn't it be simpler to just fix https://github.com/ceph/ceph/blob/master/qa/suites/rados/monthrash/thrashers/force-sync-many.yaml#L11 to use `store_thrash` and fix this comment?

Comment (batrick) [reply qa/tasks/mon_thrash.py]: agreed!

Comment (joscollin) [reply qa/tasks/mon_thrash.py]: Updated. Please check again.

**Pull request merged**